db/migrate/20220303200215_remove_support_identifier.rb

```diff
@@ -0,0 +1,7 @@
+class RemoveSupportIdentifier < ActiveRecord::Migration[5.2]
+  def change
+    if column_exists? :users, :support_identifier
+      remove_column :users, :support_identifier
+    end
```

Comment:
Not sure how this if logic interacts with Rails trying to automatically generate the reverse migration for the `change` method.

I would suggest just using `remove_column` without the `if`, remove_column by itself without a column type is not reversible.

Reply:
fixed, I probably just had this issue locally - should have reset my db first.